ChatSecure/Classes/Controllers/OTRYapViewHandler.swift

```diff
@@ -163,13 +163,13 @@ public class OTRYapViewHandler: NSObject {
             return
         }
         
-        var sectionChanges:NSArray? = nil
-        var rowChanges:NSArray? = nil
+        let sectionChanges:AutoreleasingUnsafeMutablePointer<NSArray>? = nil
```

Comment:
@davidchiles Did this get auto-suggested by the migrator? Quite the mouthful of a type :)